.editorconfig

```diff
@@ -2,9 +2,8 @@
 root = true
 
 [*]
-indent_style = tab
 charset = utf-8
-trim_trailing_whitespace = true
+end_of_line = lf
```

Comment:
@madmaxoft's comment from #3936:
>This shouldn't be handled by editorconfig, but rather by git. I want CRLFs on my Windows machine and LFs on my Linux machine.

Reply:
I had forgotten about that conversation. Let me fix the commit.